### PR TITLE
🤷‍♂️

### PR DESCRIPTION
**Pull request by frankfaustino**
- Moved some shit around.
- New order:
  - Analytics (`/pages/index.tsx`)
  - Tickets (`/pages/tickets.tsx`)
  - Assigned (`/pages/assigned.tsx`)
  - Profile (`/pages/profile.tsx`)
- Deleted RightPanel component

### CODE REVIEW

File: dashboard/server/src/communityAPI.js [hunk 1]
```diff
@@ -37,7 +37,8 @@ const resolvers = {
           title: item.title,
           body: item.body,
           createdAt: item.creationDateFormatted,
-          id: item.id
+          id: item.id,
+          totalCount: item.totalCount
```
Comment (llwor94) [inline dashboard/server/src/communityAPI.js]: wut is totalCount?

File: dashboard/client/components/SubMenu/index.tsx [hunk 2]
```diff
@@ -5,30 +5,35 @@ import React, { Fragment } from 'react'
...
+            }
+            return (
+              <Fragment key={id}>
+                <div
+                  onClick={x(id, name)}
+                  className={'sub-menu__items' + (space === id ? '-selected' : '')}
+                  key={id}
+                >
+                  <div>{name}</div>
+                  <div>{totalCount}</div>
```
Comment (llwor94) [inline dashboard/client/components/SubMenu/index.tsx]: i think u put totalCount on question ?

Comment (frankfaustino) [reply dashboard/client/components/SubMenu/index.tsx]: ya, i need to call `getTickets` on each space to calculate how many total unanswered questions there are in each space. i just didn't get around to it last night cos i fell asleep.

Screen Shot 2019-06-26 at 10 50 55 AM

is there a better way to do that?

Comment (llwor94) [reply dashboard/client/components/SubMenu/index.tsx]: yeah i can put it into the backend and jsut send it down w spaces

**Pull request merged**